src/utils/getApplicationServerSideProps.js

```diff
@@ -23,8 +23,6 @@ const getApplicationServerSideProps = (callbackFn) => async ({
 
   const queryClient = new QueryClient();
 
-  // TODO: prefetch user, permission, roles
```

Comment:
Wasn't it possible to prefetch the user, permission and roles?

Reply:
Yeah, it's possible but it adds to the loading time, a lot. It fetches with every page navigation, not just on the first hit of the application. I think it's better to leave it for now, and maybe later investigate another approach if we really want to add it.